docs/concepts/glossary.md

```diff
@@ -22,7 +22,7 @@ A field in a message which determines the type. Examples are:
 The application which maintains the ledger state.
 
 #### TRANSACTION
-The payload of a value message. It contains the particulars of a transfer of funds.
+A message with payload of type value. It contains the information of a transfer of funds.
```

Comment:
Transaction payload type is `TransactionType`

Reply:
thx i have added this info. Also in the the glossary point PAYLOAD i have added the type where applicable. 
E.g. For FPC there is a StatementType. However for faucet and the syncbeacon the name is more generic ("payloadType"). We should make the later more specific in a different PR.